[2025-08-18 18:04:02,767] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_branch_dop_operator_v3.condition scheduled__2025-08-18T21:03:00+00:00 [queued]>
[2025-08-18 18:04:02,777] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_branch_dop_operator_v3.condition scheduled__2025-08-18T21:03:00+00:00 [queued]>
[2025-08-18 18:04:02,777] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 18:04:02,777] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2025-08-18 18:04:02,777] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 18:04:02,789] {taskinstance.py:1377} INFO - Executing <Task(BranchPythonOperator): condition> on 2025-08-18 21:03:00+00:00
[2025-08-18 18:04:02,791] {standard_task_runner.py:52} INFO - Started process 2179618 to run task
[2025-08-18 18:04:02,796] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'example_branch_dop_operator_v3', 'condition', 'scheduled__2025-08-18T21:03:00+00:00', '--job-id', '4', '--raw', '--subdir', '/root/Documents/airflow/venv/lib64/python3.9/site-packages/airflow/example_dags/example_branch_python_dop_operator_3.py', '--cfg-path', '/tmp/tmp_49id__v', '--error-file', '/tmp/tmp0jzwsipj']
[2025-08-18 18:04:02,797] {standard_task_runner.py:80} INFO - Job 4: Subtask condition
[2025-08-18 18:04:02,857] {task_command.py:370} INFO - Running <TaskInstance: example_branch_dop_operator_v3.condition scheduled__2025-08-18T21:03:00+00:00 [running]> on host itoneptl
[2025-08-18 18:04:02,928] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_branch_dop_operator_v3
AIRFLOW_CTX_TASK_ID=condition
AIRFLOW_CTX_EXECUTION_DATE=2025-08-18T21:03:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-18T21:03:00+00:00
[2025-08-18 18:04:02,939] {logging_mixin.py:115} WARNING - /root/Documents/airflow/venv/lib64/python3.9/site-packages/airflow/utils/context.py:290 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-08-18 18:04:02,939] {logging_mixin.py:115} INFO - ------------- exec dttm = 2025-08-18T21:03:00+00:00 and minute = 3
[2025-08-18 18:04:02,939] {python.py:173} INFO - Done. Returned value was: empty_task_2
[2025-08-18 18:04:02,940] {skipmixin.py:140} INFO - Following branch empty_task_2
[2025-08-18 18:04:02,940] {skipmixin.py:175} INFO - Skipping tasks ['empty_task_1']
[2025-08-18 18:04:02,982] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=example_branch_dop_operator_v3, task_id=condition, execution_date=20250818T210300, start_date=20250818T210402, end_date=20250818T210402
[2025-08-18 18:04:03,007] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-18 18:04:03,040] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
